src/helpers/constants.js

```diff
@@ -1,6 +1,6 @@
-const API_URL = process.env.COSMIC_API_URL || 'https://api.cosmicjs.com'
-const UPLOAD_API_URL = process.env.UPLOAD_API_URL || 'https://upload.cosmicjs.com'
-const API_VERSION = process.env.COSMIC_API_VERSION || 'v2'
+const API_URL = process?.env?.COSMIC_API_URL ?? 'https://api.cosmicjs.com'
```

Comment:
We discussed this, and to prevent setting empty / null it should be:
```
const API_URL = process?.env?.COSMIC_API_URL || 'https://api.cosmicjs.com'
```
For reference, looks like there's widespread support for optional chaining now in most browser versions. https://caniuse.com/mdn-javascript_operators_optional_chaining

Reply:
This applies to all constants btw.